File: feature-antitheft-impl/src/main/java/com/example/antitheft/presentation/view/AntitheftMainView.java
package com.example.antitheft.presentation.view;

import moxy.MvpView;
import moxy.viewstate.strategy.OneExecutionStateStrategy;
import moxy.viewstate.strategy.StateStrategyType;

public interface AntitheftMainView extends MvpView {

    @StateStrategyType(OneExecutionStateStrategy.class)
    void showAtWork();
    @StateStrategyType(OneExecutionStateStrategy.class)
    void showAtSuccess();
    @StateStrategyType(OneExecutionStateStrategy.class)
    void showBuyWork();
    @StateStrategyType(OneExecutionStateStrategy.class)
    void showBuySuccess();

}

File: feature-antitheft-impl/src/main/java/com/example/antitheft/presentation/view/AntitheftMainView.kt
package com.example.antitheft.presentation.view

import moxy.MvpView
import moxy.viewstate.strategy.OneExecutionStateStrategy
import moxy.viewstate.strategy.StateStrategyType
import moxy.viewstate.strategy.alias.OneExecution

@OneExecution
interface AntitheftMainView : MvpView {

    fun showAtWork()

    fun showAtSuccess()

    fun showBuyWork()

    fun showBuySuccess()
}